src/main/java/schemadefiner/retrieveschema/JdbcCrawler.java

```diff
@@ -0,0 +1,62 @@
+package schemadefiner.retrieveschema;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import schemacrawler.crawl.StreamlinerCatalog;
+import schemacrawler.schema.Catalog;
+import schemacrawler.schema.Schema;
+import schemacrawler.schema.Table;
+import schemacrawler.schemacrawler.SchemaCrawlerException;
+import schemacrawler.schemacrawler.SchemaCrawlerOptions;
+import schemacrawler.tools.utility.SchemaCrawlerUtility;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Supplier;
+
+public class JdbcCrawler implements SchemaDefiner {
+
+    private static final Logger log = LoggerFactory.getLogger(JdbcCrawler.class);
+
+    private String user;
+    private String password;
+    private String jdbcUrl;
+    private final Supplier<Connection> connectionSupplier;
+    private final SchemaCrawlerOptions schemaCrawlerOptions;
+
+    public JdbcCrawler(String jdbcUrl, Supplier<Connection> connectionSupplier, SchemaCrawlerOptions schemaCrawlerOptions) {
+        this.jdbcUrl = jdbcUrl;
+        this.connectionSupplier = connectionSupplier;
+        this.schemaCrawlerOptions = schemaCrawlerOptions;
+    }
+
+    @Override
+    public StreamlinerCatalog retrieveSchema() {
+        if (jdbcUrl.startsWith("jdbc:oracle")) {
+            log.info("ByPassing Oracle from JdbcCrawler");
+            return null;
```

Comment:
Reminder this part needs to be implemented. Should be a straight copy from the existing implementation